### DISCUSSION TITLE
Version of libjpeg-9__.dll?

### DISCUSSION THREAD

**Discussion by greggmorris** [StrawberryPerl/Perl-Dist-Strawberry#129] [Q&A] [answered]
Is it possible to determine what version of libjpeg-9__.dll is included with Strawberry Perl 5.38? I can't find any version information in the file itself. The latest version on ijg.org is 9e, released in January, 2022. Thanks for any help.

**Comment by shawnlaffan** [comment, marked as answer]
It's 9c.  

This is in the license file at ```...\licenses\libjpeg\_INFO_```.  

The build script used also includes the version number, although that's a far from obvious place to go looking.  
https://github.com/StrawberryPerl/build-extlibs/blob/56e3a3b834962b7cdcb459247b30514d213207ee/5034#L39

I also ran pkg-config but the resulting version is non-standard.  Possibly there is an issue in the build process but this is the same for SP 5.30.0 so I assume others as well.
```
pkg-config libjpeg --modversion
12:0:3
```

**Reply by greggmorris** [reply]
Thanks, now I know where to look for this information in the future.

**Reply by shawnlaffan** [reply]
Not a problem.  

We should probably update libjpeg to 9e in future versions.
